### PR TITLE
DF-237 Removed note from tools and utilities guide

### PR DESCRIPTION
**Pull request by sheet20**
## What Changed?

- DF-237: Removed note in Tools and Utilities Guide.
- Title change in v11 has impact in 13, 12 versions. Updated the titles to keep it consistent across all versions. 
(this has further impact in all versions, which I'll update as and when I've bandwidth).

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**
- [ ] This PR adds new content
- [x] This PR changes existing content
- [ ] This PR removes existing content

### CODE REVIEW

**Comment by sheet20** [comment]
Merge with develop branch

**Review by drothery-edb — APPROVED** [review]

File: product_docs/docs/epas/12/epas_compat_tools_guide/index.mdx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 navTitle: Tools and Utilities Guide
-title: "Database Compatibility for Oracle Developer’s Tools and Utilities Guide"
+title: "Database Compatibility for Oracle Developer Tools and Utilities Guide"
```
Comment (drothery-edb) [inline product_docs/docs/epas/12/epas_compat_tools_guide/index.mdx:3]: make plural

File: product_docs/docs/epas/13/epas_compat_tools_guide/index.mdx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 navTitle: Tools and Utilities Guide
-title: "Database Compatibility for Oracle Developer’s Tools and Utilities Guide"
+title: "Database Compatibility for Oracle Developer Tools and Utilities Guide"
```
Comment (drothery-edb) [inline product_docs/docs/epas/13/epas_compat_tools_guide/index.mdx:3]: make plural

**Review by drothery-edb — COMMENTED** [review]
review comments

File: product_docs/docs/epas/13/epas_compat_tools_guide/index.mdx (hunk 2 above)
Comment (sheet20) [reply product_docs/docs/epas/13/epas_compat_tools_guide/index.mdx:3]: change made

**Pull request merged**